UPGRADE-6.4.md

```diff
@@ -29,6 +29,7 @@ Form
 
  * Deprecate using `DateTime` or `DateTimeImmutable` model data with a different timezone than configured with the
    `model_timezone` option in `DateType`, `DateTimeType`, and `TimeType`
+ * Deprecate `PostSetDataEvent::setData()` and `PostSubmitEvent::setData()`, these methods will throw an exception starting with 7.0
```

Comment:
let's give the solution instead of the new (useless) behavior; something like
```suggestion
 * Deprecate `PostSetDataEvent::setData()` and `PostSubmitEvent::setData()`, use `form.pre_set_data" instead` instead
```

Reply:
Done.